### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by sreya-kata**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,111 @@
...
 
-1.  **Open the source code and start editing!**
+In the project directory is a file called `env_example`. This holds all of the necessary environment variables to create this project, including the Contentful access token.
 
-    Your site is now running at `http://localhost:8000`!
+Copy the file twice, name the new files .env.development and .env.production, and add the Contentful API keys to the variables. You can find these API keys in your Contentful space under Settings -> API keys -> vital-site. You only need to use Space ID and Content Delivery API - access token. Do **not** commit these to the Github repo
 
-    _Note: You'll also see a second link: _`http://localhost:8000/___graphql`_. This is a tool you can use to experiment with querying your data. Learn more about using this tool in the [Gatsby tutorial](https://www.gatsbyjs.com/tutorial/part-five/#introducing-graphiql)._
+```shell
+cp env_example .env.development
```
Comment (josephmannis) [inline README.md]: This doesn't match Ryan's PR, which names the file as `.env.example`

File: README.md [hunk 2]
```diff
@@ -1,46 +1,111 @@
...
+
+```shell
+# Inside the project directory
+npm install
+```
+
+Run server/development environment:
+
+```shell
+npm run dev
```
Comment (josephmannis) [inline README.md]: This doesn't appear to be a command, I think it should be `develop` (or you can change it to `dev` in `package.json`)

**Review by josephmannis — CHANGES_REQUESTED** [review]
Couple of little fixes, but otherwise very descriptive and nice.

**Comment by sreya-kata** [comment]
Made changes and fixed netlify hyperlinks

**Review by sreya-kata — COMMENTED** [review]
Made changes and fixed netlify hyperlinks

**Review by josephmannis — APPROVED** [review]
🚀

**Pull request merged**